### PR TITLE
chore: add examples folder and two example scripts

### PR DESCRIPTION
**Pull request by chaseeb**
#### Problem

the web3 repo has no examples for how to use it and developers have been asking for them

#### Summary of Changes

created examples folder and added two example scripts for sending sol and minting tokens

Fixes #

### CODE REVIEW

**Comment by t-nelson** [comment]
Ah nice!  These will be much nicer to refer to than, "Uhhh, go look at the tests?"

I think the token example belongs in the [`@solana/spl-token`](https://github.com/solana-labs/solana-program-library/tree/master/token/js).  Mind moving it there?

File: web3.js/examples/send_sol.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import * as web3 from '@solana/web3.js';
+
+(async () => {
+  // Connect to cluster
+  var connection = new web3.Connection(web3.clusterApiUrl('devnet'));
+
+  // Generate a new random public key
+  var from = web3.Keypair.generate();
+  await connection.requestAirdrop(from.publicKey, 1000000000);
```
Comment (jstarry) [inline web3.js/examples/send_sol.js:9]:
```suggestion
  var airdropSignature = await connection.requestAirdrop(from.publicKey, web3.LAMPORTS_PER_SOL);
  await connection.confirmTransaction(airdropSignature);
```

File: web3.js/examples/send_sol.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  // Generate a new random public key
+  var to = web3.Keypair.generate();
+  await connection.requestAirdrop(to.publicKey, 1000000000);
```
Comment (jstarry) [inline web3.js/examples/send_sol.js:13]:
```suggestion
```

File: web3.js/examples/send_sol.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    // Sign transaction, broadcast, and confirm
+    var signature = await web3.sendAndConfirmTransaction(
+      connection,
+      transaction,
+      [from],
+      {commitment: 'confirmed'},
+    );
+    console.log('SIGNATURE', signature);
+  }, 1000);
```
Comment (jstarry) [inline web3.js/examples/send_sol.js:34]:
```suggestion
    // Add transfer instruction to transaction
    var transaction = new web3.Transaction().add(
      web3.SystemProgram.transfer({
        fromPubkey: from.publicKey,
        toPubkey: to.publicKey,
        lamports: web3.LAMPORTS_PER_SOL / 100,
      }),
    );

    // Sign transaction, broadcast, and confirm
    var signature = await web3.sendAndConfirmTransaction(
      connection,
      transaction,
      [from],
    );
    console.log('SIGNATURE', signature);
```

File: web3.js/examples/send_sol.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import * as web3 from '@solana/web3.js';
+
+(async () => {
+  // Connect to cluster
+  var connection = new web3.Connection(web3.clusterApiUrl('devnet'));
```
Comment (jstarry) [inline web3.js/examples/send_sol.js:5]:
```suggestion
  var connection = new web3.Connection(web3.clusterApiUrl('devnet'), 'confirmed');
```

**Comment by jstarry** [comment]
Thanks for adding this @chaseeb!

**Review by jstarry — APPROVED** [review]

**Pull request merged**